addons/mail/static/src/components/follower_list_menu/follower_list_menu.xml

```diff
@@ -12,13 +12,13 @@
 
                     <t t-if="state.isDropdownOpen">
                         <div class="o_FollowerListMenu_dropdown dropdown-menu dropdown-menu-right" role="menu">
-                            <t t-if="thread.channel_type !== 'chat'">
+                            <t t-if="thread.channel_type !== 'chat' and thread.hasWriteAccess">
                                 <a class="o_FollowerListMenu_addFollowersButton dropdown-item" href="#" role="menuitem" t-on-click="_onClickAddFollowers">
                                     Add Followers
                                 </a>
                             </t>
                             <t t-if="thread.followers.length > 0">
-                                <div role="separator" class="dropdown-divider"/>
+                                <div t-if="thread.hasWriteAccess" role="separator" class="dropdown-divider"/>
```

Comment:
Probably should move the separator into a separate condition above the parent `t-if` that includes both the condition on write but also channel_type/channel, because this seems incomplete.